src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -82,6 +82,14 @@ class LocationAttribute(
   @Override
   override fun toString(): String = this::class.simpleName +
     "(locationAttributeId = $locationAttributeId, prisonId = ${prison.prisonId}, dpsLocationId = $dpsLocationId)"
+
+  // TODO not finished
+  fun isAvailableFor(probationTeam: ProbationTeam, onDateTime: LocalDateTime) = (locationUsage == LocationUsage.PROBATION && partyIsAllowed(probationTeam.code)) || locationUsage == LocationUsage.SHARED
```

Comment:
Not used in the service layer yet but I was thinking the logic for checking if an attribute applies can be pushed into the actual attribute.